events.go

```diff
@@ -309,6 +309,11 @@ type MessageReactionRemoveAll struct {
 	*MessageReaction
 }
 
+// MessageReactionRemoveEmoji is the data for a MessageReactionRemoveEmoji event.
+type MessageReactionRemoveEmoji struct {
+	*MessageReaction
```

Comment:
Since we're embedding `MessageReaction`, I think it would be a good idea to mention in the struct that `UserID` will not be present.
Like this:
```diff
+ // NOTE: will not be present in MessageReactionRemoveEmoji event
```
And add `omitempty` to it.